eggs.ts

```diff
@@ -29,4 +29,5 @@ publish({
     `./src/*.node.ts`,
   ],
   yes: true,
-}, pkg.name.replace('@satont/', ''));
\ No newline at end of file
+  dryRun: Deno.env.get('DRY') === 'true' ? true : false,
+}, `grammy-${pkg.name.replace('@grammyjs/', '').split('-').reverse().join('-')}`);
```

Comment:
This clearly isn't pretty, but it works and I guess it's a small price to pay if it means we don't have to re-publish the packages.

Reply:
>This clearly isn't pretty

I absolutely agree. But I don't see any other solution, unless you put a file somewhere near only to publish on nest.land and read the name from there.

> but it works and I guess it's a small price to pay if it means we don't have to re-publish the packages.

Yes, republish doesnt needed. Also nest.land doesnt bind to repository, and it can be swaped.